safe/gui/tools/wizard_strings.py

```diff
@@ -19,23 +19,24 @@
 
 category_question = tr(
     'By following the simple steps in this wizard, you can assign '
-    'keywords to your layer: <b>%s</b>. First you need to define '
-    'the purpose of your layer.')  # (layer name)
+    'keywords to your layer: <b>%s</b>. First you need to define the purpose '
+    'of your layer. Is it a <b>hazard</b>, <b>exposure</b>, or '
+    '<b>aggregation</b> layer? ')  # (layer name)
 category_question_hazard = tr(
     'You have selected a layer that needs to have keywords assigned or '
     'updated. In the next steps you can assign keywords to that layer. '
-    'First you need to confirm the layer represents a hazard.')
+    'First you need to confirm the layer is a <b>hazard</b> layer.')
 category_question_exposure = tr(
     'You have selected a layer that needs to have keywords assigned or '
     'updated. In the next steps you can assign keywords to that layer. '
-    'First you need to confirm the layer represents an exposure.')
+    'First you need to confirm the layer is an <b>exposure</b>.')
```

Comment:
First you need to confirm the layer is an <b>exposure</b> layer.

(for consistency with previous paragraph?)